src/components/TopMovie.js

```diff
@@ -8,12 +8,12 @@ const TopMovie = ({ movie }) => {
     <>
     <article className="top-movie">
       <h3><Link to={{
-        pathname:`/movie-details/${movie.title}`,
+        pathname:`/movie-details/${movie.title.replace(/\s+/g, '')}`,
```

Comment:
Love this! Now the endpoint is much more clear and more specific for what page the user is on!

Reply:
Seconded. Much more user friendly to have the endpoint be the title rather than the id